src/main/java/uk/gov/dvsa/mot/fixtures/mot/TesterDoesStepDefinitions.java

```diff
@@ -396,6 +396,24 @@ public TesterDoesStepDefinitions(WebDriverWrapper driverWrapper) {
         And("^I check the advisory section of the test summary has \"([^\"]+)\"$", (String text) ->
                 assertTrue(driverWrapper.getTextFromDefinitionList("Advisory text").contains(text)));
 
+        And("^I check the dangerous failures section of the brake test summary of the test summary has \"([^\"]+)\"$",
+                (String text) -> assertTrue(driverWrapper.getTextFromDefinitionListOfIndex("Dangerous failures", 1)
+                        .contains(text)));
+
+        And("^I check the major failures section of the brake test summary of the test summary has \"([^\"]+)\"$",
+                (String text) -> assertTrue(driverWrapper.getTextFromDefinitionListOfIndex("Major failures", 1)
+                        .contains(text)));
+
+        And("^I check the dangerous failures section of the brake test summary of the test summary "
+                        + "does not have \"([^\"]+)\"$",
+                (String text) -> assertFalse(driverWrapper.getTextFromDefinitionListOfIndex("Dangerous failures", 1)
+                        .contains(text)));
+
+        And("^I check the major failures section of the brake test summary of the test summary "
+                        + "does not have \"([^\"]+)\"$",
+                (String text) -> assertFalse(driverWrapper.getTextFromDefinitionListOfIndex("Major failures", 1)
+                        .contains(text)));
+
```

Comment:
These new step definitions will need adding to docs/step-definitions.md

Reply:
Added these :)